pyrometheus/codegen/python.py

```diff
@@ -304,6 +306,20 @@ def get_mixture_internal_energy_mass(self, temperature, mass_fractions):
         emix = self.get_mass_average_property(mass_fractions, e0_rt)
         return self.gas_constant * temperature * emix
 
+    def get_specific_heat_ratio(self, temperature, mass_fractions):
+        cp0_r = self.get_species_specific_heats_r(temperature)
+        cpmix = self.get_mass_average_property(mass_fractions, cp0_r)
+        sum_Yk_Wk = sum([
```

Comment:
This is the reciprocal of the mixture molecular weight. Can we name it `inv_mix_mol_weight` to be more descriptive? It will also dispel some of the issues with Flake8 (variables need to be all lower case).